qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -269,30 +269,40 @@ def _check_duration(duration: Optional[float] = None) -> float:
 
 
 def new_ramsey_sequence(duration=None, pre_post_rotation=False, **kwargs):
-    """
-    Creates Ramsey sequence.
+    r"""
+    Creates the Ramsey sequence.
 
     Parameters
     ----------
     duration : float, optional
-        Total duration of the sequence. Defaults to None
+        Total duration of the sequence :math:`\tau`. Defaults to None.
     pre_post_rotation : bool, optional
-        If True, a :math:`X_{\\pi.2}` rotation
+        If True, a :math:`X_{\pi / 2}` rotation
         is added at the start and end of the sequence.
     kwargs : dict
-        Additional keywords required by
-        qctrlopencontrols.sequences.DynamicDecouplingSequence
+        Additional keywords required by DynamicDecouplingSequence.
 
     Returns
     -------
-    qctrlopencontrols.dynamic_decoupling_sequences.DynamicDecouplingSequence
-        The Ramsey sequence
+    DynamicDecouplingSequence
+        The Ramsey sequence.
 
     Raises
     ------
     ArgumentsValueError
         Raised when an argument is invalid.
 
+    Notes
+    -----
+    Technically, the Ramsey sequence [#]_ does not decouple the system from the environment.
```

Comment:
Interesting, I didn't know (or forgot) that `[#]_` worked in reST

Reply:
I steal this from Harry's previous PR.